examples/groth16-verifier/src/lib.rs

```diff
@@ -0,0 +1,186 @@
+// Copyright 2023 RISC Zero, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use anyhow::Result;
+use bonsai_sdk::alpha as bonsai_sdk;
+use byteorder::{BigEndian, WriteBytesExt};
+use hello_world_methods::{MULTIPLY_ELF, MULTIPLY_ID};
+use risc0_zkvm::{
+    default_prover, groth16::Groth16Seal, serde::to_vec, ExecutorEnv, Groth16Receipt, MemoryImage,
+    Program, Receipt, GUEST_MAX_MEM, PAGE_SIZE,
+};
+use std::time::Duration;
+
+// Run Bonsai for the hello world example
+pub fn run_bonsai(input_data: Vec<u8>) -> Result<Receipt> {
```

Comment:
I think for the initial example, we should leave Bonsai out of it. This is because I think we need to unify and integrate the new Groth16 receipt into Bonsai and then hide all these details behind the `Prover` trait that zkvm already exposes.

Reply:
I think we can still have a test somewhere that does the verification. Perhaps we don't need an example until it's all integrated with Bonsai.